texttospeech/ssml_addresses/tts.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# [START tts_ssml_address_imports]
+from google.cloud import texttospeech
+
+# For Python 3, instead use:
+# import html
+import cgi
+# [END tts_ssml_address_imports]
+
+
+# [START tts_ssml_address_audio]
+def ssml_to_audio(ssml_text, outfile):
+    # Generates SSML text from plaintext.
+    #
+    # Given a string of SSML text and an output file name, this function
+    # calls the Text-to-Speech API. The API returns a synthetic audio
+    # version of the text, formatted according to the SSML commands. This
+    # function saves the synthetic audio to the designated output file.
+    #
+    # Args:
+    # ssml_text: string of SSML text
+    # outfile: string name of file under which to save audio output
+    #
+    # Returns:
+    # nothing
+
+    # Checks to make sure ssml_text is not empty
+    if ssml_text is None:
```

Comment:
nit: Could probably remove this check (plus the part of test_ssml_to_audio that tests this) can assume they'll at least provide a non-empty string as part of your args definition above.